### DISCUSSION TITLE
Filament relationship with casted titleAttribute

### DISCUSSION THREAD

**Discussion by spiritkiddie** [filamentphp/filament#14001] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

SelectFilter::make(name: 'finance')->relationship(name: 'finance', titleAttribute: 'creditScore') 
The title attribute "creditScore" field is a casted enum field which throws error  "Argument #2 ($label) must be of type string enum given"

**Comment by e2d2-dev** [comment]
Hi,

maybe You should try to build Your own filter and use the map() function on the query.

[https://filamentphp.com/docs/3.x/tables/filters/custom](url)
